src/app.ts

```diff
@@ -61,6 +69,13 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 
+app.use((req, res, next) => {
```

Comment:
Should this new code be wrapped in a feature flag?

Reply:
No, it does not need to be because that bit of code is feature-flag agnostic. Having the feature-flag present doesn't break any flow, but it's not required.

Happy to discuss this (and any of the issues raised above) if you believe I might have overlooked something important.

Cheers.

Reply:
We've done something similar (I think) in the `journey.detection.middleware` but there we use `res.locals` to get the value available in all the Nunjucks templates.

Is `addGlobal` better to use here?

Reply:
Yes, I believe `addGlobal` is a better approach here so as to keep the data export consistent with the chunk of export code a few lines prior.